dts/bindings/counter/nxp,lptmr.yaml

```
@@ -50,15 +50,19 @@ properties:
 
   prescale-glitch-filter:
     type: int
-    default: 1
-    enum: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
+    default: 0
+    enum: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
     description: |
-      When in prescaler mode, the counter is incremented every
+      When in prescaler mode, if prescale-glitch-filter is 0, the prescaler
+        is bypassed, the selected prescaler clock increments Counter (CNR) on every
+        clock cycle. Otherwise the counter is incremented every
         2 ^ [prescaler-glitch-filter] clock cycles.
-      When in pulse mode, the counter is incremented every
-        2 ^ [prescaler-glitch-filter] rising edges detected
-        by the pin configured from the input-pin value.
-        Note, that the pulse mode cannot be 2 ^ 16.
+      When in pulse mode, if prescale-glitch-filter is 0, the glitch filter
+        is bypassed, the selected input source increments the value of Counter(CNR)
+        every time it asserts. Otherwise the counter is incremented every
+        2 ^ [prescaler-glitch-filter] rising edges detected by the pin configured
+        from the input-pin value.
+      Note, that the pulse mode cannot be 2 ^ 16.
 
```

Should we deprecate the prescaler here?
```suggestion
  # Keep for DT ABI, but no longer used by the driver
  prescaler:
    type: int
    deprecated: true
    description: Deprecated; use prescale-glitch-filter instead.
```